src/plugin/localeData/index.js

```diff
@@ -23,7 +25,8 @@ export default (o, c, dayjs) => { // locale needed later
         (instance ? instance.format('dd') : getShort(this, 'weekdaysMin', 'weekdays', 2)),
       weekdaysShort: instance =>
         (instance ? instance.format('ddd') : getShort(this, 'weekdaysShort', 'weekdays', 3)),
-      longDateFormat: format => this.$locale().formats[format]
+      longDateFormat: format =>
+        this.$locale().formats[format] || t(this.$locale().formats[format.toUpperCase()])
```

Comment:
can we make this a separate function like
```
const getLongDateFormat = (locale) => locale.formats[format] || t(locale.formats[format.toUpperCase()])
```

Reply:
I did
```javascript
const getLongDateFormat = format =>
  getDayjsLocaleObject().formats[format] ||
  t(getDayjsLocaleObject().formats[format.toUpperCase()])
```